app/templates/notifications.hbs

```diff
@@ -0,0 +1,10 @@
+<h1 class="ui header">{{t 'Notifications'}}</h1>
+<div class="ui divider"></div>
+<div class="row">
+  <a href="/notifications" class="ui button left attached">{{t 'Unread'}}</a>
```

Comment:
@utkarshnath Great work! Can you please check the unread & all buttons? The text seems to be misaligned.
![screenshot from 2017-06-02 09-45-46](https://cloud.githubusercontent.com/assets/12807846/26717520/590b6d4e-479b-11e7-8ded-4f24a8057436.png)